### PR TITLE
Dmnt styling

### PR DESCRIPTION
**Pull request by DeeThompson**
We had two issues(outlined in the acceptance criteria). We decided to complete the tasks for this final week as one group. The styling to both the add item and detail screen was implemented while working through this cohort. However with week 7 and 8 there were additional styling updates added. Tailwind CSS was installed, a background image was added to the list-view and add-item pages. Also the user is able to join and existing shopping list or is prompted to create a new one. 

## Acceptance Criteria

* The add an item screen (the screen a user uses to add a new item to their list) looks something like the following (or equivalent, use your creativity!):

add-item

* The item detail screen (the screen a user sees when they tap on one of their items in the shopping list) looks something like the following (or equivalent, use your creativity!):

item-detail

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓  | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

item-screen before
</br></br>
add-item-sceen-before

### After

item-detail-screen-after
</br></br>
add-item-screen-after

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const tailwindcss = require('tailwindcss');
+module.exports = {
+    plugins: [
+        tailwindcss('./tailwind.js'),
+        require('autoprefixer')
+    ],
+};
```
Comment (alejandronanez) [inline postcss.config.js:7]: I'm glad you had the change to play with different tools :)

**Review by alejandronanez — APPROVED** [review]
This is looking great y'all thanks for the hard work!

Just one tiny thing to remember for future projects: Ideally you should have only one CSS Library/Framework (Tailwind vs Bootstrap). That said, if you're moving towards Tailwind, I'd suggest remove any Bootstrap reference.

Open questions for the demo today: Is Tailwind what you expected? Would you use it again?

Again, great work! 👏

File: src/Components/shareList.js [hunk 2]
```diff
@@ -34,7 +33,7 @@ export function ShareList() {
   };
 
   return (
-    <Form onSubmit={handleSubmitShareCode} className="pa2">
+    <form onSubmit={handleSubmitShareCode} className="pa2">
```
Comment (msholty-fd) [inline src/Components/shareList.js]: I like that you went with the native `form` element here. This makes this component that much less complex.

**Review by msholty-fd — APPROVED** [review]
I really like the direction y'all went with this, and I was particularly interested to see how you used tailwind. I agree with @alejandronanez that you should remove the usages of bootstrap in this app since tailwind should give you everything you need. Other than that, looks awesome! 🏆

**Pull request merged**